### ISSUE TITLE
A11y_AzureDataStudioSQL_PublishProjects_Generate Scripts(Terminal)_Zoom: Data is not properly visible under Task(terminal)at 200% zoom

### ISSUE THREAD

**Issue by Shivamd1** [microsoft/azuredatastudio#14517]
“[Check out Accessibility Insights!](https://accessibilityinsights.io/) - Identify accessibility bugs before check-in and make bug fixing faster and easier.”

**GitHubTags:** #A11yWCAG2.1;#A11yTCS;#A11ySev2;#SH_AzureDataStudioSQL_Feb2021;#SQL Azure Data Studio;#Win10;#FTP;#DesktopApp;#WCAG1.4.4;#AINotInScope;#Zoom;

### Prerequisite: 

- Set the text size to 200% in "Ease of access settings"

### Environment Details:
Application Name: Azure Data Studio
Windows Version: Win10

### Repro Steps:

1. Launch the application-Azure Database Project and install extension SQL Database Projects.
2. Tab till SQL Database Project Home Screen with heading "Projects".
3. Tab to connections select your server and connect to it.
4. Tab to Projects tab an press shit+f10 on any created project.
5. Tab till "Publish Project" and hit enter
6. Hit on Generate script and navigate to task
7. Check if data is properly visible under task

### Actual:
Data is not properly visible under Task(terminal)at 200% zoom

### Expected:
All the data should be properly visible at 200% zoom.

### User Impact:
Low Vision users will face difficulty in getting the data.

**Comment by kisantia** [comment]
@Shivamd1 can you share a screenshot of how this looks? What is the expected behavior?

I tried zooming to 200% and saw that tasks look like this, but the tooltip shows the full text
![image](https://user-images.githubusercontent.com/31145923/110055329-a4ef6300-7d11-11eb-882e-91072f60285a.png)

**Comment by kisantia** [comment]
closing this since the original issue is resolved and fix has been verified. Please open a new issue for making the horizontal scrolling keyboard accessible, as this problem is also present in the connections view and file explorer view.

**Comment by fsteffi** [comment]
issue fixed in

githubtags:#Closed;

Version: 1.43.0-insider (user setup)
Commit: b4d5a08199e9e0cb0138ce540c086ac32d072019
Date: 2023-03-07T06:31:08.002Z
VS Code: 1.67.0
Electron: 19.1.8
Chromium: 102.0.5005.167
Node.js: 16.14.2
V8: 10.2.154.15-electron.0
OS: Windows_NT x64 10.0.25300

after collapsing the side panel
![MicrosoftTeams-image (1)](https://user-images.githubusercontent.com/24580570/227188966-b1aa4825-105e-405a-94e9-399447b7ef44.png)
